src/OpenTelemetry/Trace/BatchExportActivityProcessorOptions.cs

```diff
@@ -78,13 +83,7 @@ private static bool TryLoadEnvVarInt(string envVarKey, out int result)
                 return false;
```

Comment:
Do we want to try parsing a whitespace only string if the value should be an `int`?

With the current code `OTEL_BSP_EXPORT_TIMEOUT=   ` it will throw an exception
If we change to `string.IsNullOrWhiteSpace` then we will act as if the environment variable is not set at all.

Reply:
`OTEL_BSP_EXPORT_TIMEOUT= ` is working as in all of the shells I know it will set an empty string. It is `OTEL_BSP_EXPORT_TIMEOUT=" "` which would not. And it is such a corner case that personally I would prefer to treat as error.